README.md

```diff
@@ -212,79 +244,51 @@ julia> pairwise(Euclidean(1e-12), x, x)
+specialized implementations provided in *Distances.jl*. The task in each iteration
+is to compute a specific distance in a pairwise manner between columns in a
+`100-by-200` and `100-by-250` matrices, which will result in a `200-by-250`
+distance matrix.
 
 |  distance  |  loop  |  pairwise  |  gain  |
 |----------- | -------| ----------| -------|
-| SqEuclidean | 0.012498s |  0.000170s | **73.6596** |
-| Euclidean | 0.012583s |  0.000257s | 48.9628 |
-| PeriodicEuclidean | 0.030935s |  0.017572s |  1.7605 |
-| Cityblock | 0.012416s |  0.000910s | 13.6464 |
-| TotalVariation | 0.012763s |  0.000959s | 13.3080 |
-| Chebyshev | 0.023800s |  0.012042s |  1.9763 |
-| Minkowski | 0.121388s |  0.107333s |  1.1310 |
-| Hamming | 0.012171s |  0.000689s | 17.6538 |
-| CosineDist | 0.017474s |  0.000214s | **81.6546** |
-| CorrDist | 0.028195s |  0.000259s | **108.7360** |
-| ChiSqDist | 0.014372s |  0.003129s |  4.5932 |
-| KLDivergence | 0.079669s |  0.063491s |  1.2548 |
-| RenyiDivergence | 0.134093s |  0.117737s |  1.1389 |
-| RenyiDivergence | 0.047658s |  0.024960s |  1.9094 |
-| JSDivergence | 0.121999s |  0.110984s |  1.0993 |
-| BhattacharyyaDist | 0.021788s |  0.009414s |  2.3145 |
-| HellingerDist | 0.020735s |  0.008784s |  2.3606 |
-| WeightedSqEuclidean | 0.012671s |  0.000186s | **68.0345** |
-| WeightedEuclidean | 0.012867s |  0.000276s | **46.6634** |
-| WeightedCityblock | 0.012803s |  0.001539s |  8.3200 |
-| WeightedMinkowski | 0.127386s |  0.107257s |  1.1877 |
-| WeightedHamming | 0.012240s |  0.001462s |  8.3747 |
-| SqMahalanobis | 0.214285s |  0.000330s | **650.0722** |
-| Mahalanobis | 0.197294s |  0.000420s | **470.2354** |
-| BrayCurtis | 0.012872s |  0.001489s |  8.6456 |
-
-For distances of which a major part of the computation is a quadratic form (e.g. *Euclidean*, *CosineDist*, *Mahalanobis*), the performance can be drastically improved by restructuring the computation and delegating the core part to `GEMM` in *BLAS*. The use of this strategy can easily lead to 100x performance gain over simple loops (see the highlighted part of the table above).
+| SqEuclidean | 0.001273s |  0.000124s | **10.2290** |
+| Euclidean | 0.001445s |  0.000194s |  **7.4529** |
+| CosineDist | 0.001928s |  0.000149s | **12.9543** |
+| CorrDist | 0.016837s |  0.000187s | **90.1854** |
+| WeightedSqEuclidean | 0.001603s |  0.000143s | **11.2119** |
+| WeightedEuclidean | 0.001811s |  0.000238s |  **7.6032** |
+| SqMahalanobis | 0.308990s |  0.000248s | **1248.1892** |
+| Mahalanobis | 0.313415s |  0.000346s | **906.1836** |
+
+For distances of which a major part of the computation is a quadratic form
+(e.g. *Euclidean*, *CosineDist*, *[Sq]Mahalanobis*), the performance can be
+drastically improved by restructuring the computation and delegating the core
+part to `GEMM` in *BLAS*. The use of this strategy can lead to dramatic
+performance gain over simple loops; see the the table above.
```

Comment:
It would be more logical to put this before the table IMO. And again mention that for other distances the naive approach is used.